specification/splitio/resource-manager/readme.md

```diff
@@ -0,0 +1,51 @@
+# Split.IO
+
+> see https://aka.ms/autorest
+
+This is the AutoRest configuration file for Split.IO.
+
+## Getting Started
+
+To build the SDK for Split.IO, simply [Install AutoRest](https://aka.ms/autorest/install) and in this folder, run:
+
+> `autorest readme.md`
+
+To see additional help and options, run:
+
+> `autorest --help`
+
+For other options on installation see [Installing AutoRest](https://aka.ms/autorest/install) on the AutoRest github page.
+
+---
+
+## Configuration
+
+### Basic Information
+
+These are the global settings for the Split.IO.
+
+```yaml
+openapi-type: arm
+openapi-subtype: rpaas
+tag: package-2024-03-01-preview
+```
+
+### Tag: package-2024-03-01-preview
+
+These settings apply only when `--tag=package-2024-03-01-preview` is specified on the command line.
+
+```yaml $(tag) == 'package-2024-03-01-preview'
+input-file:
+  - SplitIO.Experimentation/preview/2024-03-01-preview/splitio.json
+```
+# Code Generation
+
+## Suppression
+``` yaml
+directive:
+  - suppress: PathResourceProviderNamePascalCase
+    from: splitio.json
+    reason: This is a specific resource provider name requested by liftr partner.
+  - suppress: PatchBodyParametersSchema
```

Comment:
>- suppress: PatchBodyParametersSchema [](http://example.com/codeflow?start=2&length=37)

could you include "where" that mentions the path , instead of suppressing for the whole file ?

Reply:
ok, will do

Reply:
I tried to include "where" in suppression in different ways but checks error cannot be resolved in those cases. I think it's because what's flagged ("Sku" name is a required property and "ManagedServiceIdentity" type is a required property) is in common-type files. Is there a way suppression can point to common-type files?